dbms/src/Interpreters/Context.cpp

```diff
@@ -1866,6 +1866,28 @@ Context::SampleBlockCache & Context::getSampleBlockCache() const
 }
 
 
+bool Context::checkEmptyParamSubstitution() const
```

Comment:
We should rename "Param" to "Parameter" everywhere in code,
because I don't like abbreviations.